README.md

```diff
@@ -28,7 +28,7 @@ and treat as best.
 ## Collator
 
 A Polkadot [collator](https://wiki.polkadot.network/docs/en/learn-collator) for the parachain is
-implemented by [`cumulus-collator`](collator).
+implemented by [`cumulus-collator`](https://github.com/paritytech/cumulus/tree/master/client/collator).
```

Comment:
I think this was maybe more about the `polkadot-collator` binary.

Reply:
```suggestion
implemented by the `polkadot-collator` binary.
```